assets/models/system/openai-clip-image-text-embeddings-vit-large-patch14-336/description.md

```diff
@@ -0,0 +1,140 @@
+
+# Evaluation Results
+This model was evaluated for text retrieval and image retrieval tasks on the Flickr30k and MSCOCO datasets. The results from Table 13 in the [original CLIP paper](https://arxiv.org/pdf/2103.00020.pdf) are summarized below
+
+Text Retrieval
+|Dataset| R@1 | R@5 |
+|-------|-----|-----|
+| Flickr30k| 88.0 | 98.7 |
+| MSCOCO | 58.4 | 81.5 |
+
+Image Retrieval
+|Dataset| R@1 | R@5 |
+|-------|-----|-----|
+| Flickr30k| 68.7 | 90.6 |
+| MSCOCO | 37.8 | 62.4 |
+
+# Limitations and Biases
+
+## Bias
+The authors of the [original CLIP paper](https://arxiv.org/pdf/2103.00020.pdf) found that the performance of the model and its biases can depend significantly on class design and the choices one makes for categories to include and exclude. They tested the risk of certain kinds of denigration with CLIP by classifying images of people from Fairface into crime-related and non-human animal categories. They found significant disparities with respect to race and gender, which could shift based on how the classes were constructed. The authors also tested the performance of CLIP on gender, race, and age classification using the Fairface dataset. They found that the accuracy for gender classification was greater than 96% across all races, with 'Middle Eastern' having the highest accuracy (98.4%) and 'White' having the lowest (96.5%). Additionally, CLIP averaged ~93% for racial classification and ~63% for age classification.
+
+# License
+The MIT License is a permissive free software license originating at the Massachusetts Institute of Technology (MIT). The license allows users to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the software, subject to the conditions that the copyright notice and permission notice appear in all copies or substantial portions of the software.
+
+
+### Inference samples
+
+Inference type|Python sample (Notebook)|CLI with YAML
+|--|--|--|
+Real time|<a href="https://aka.ms/azureml-infer-sdk-image-text-embeddings" target="_blank">image-text-embeddings-online-endpoint.ipynb</a>|<a href="https://aka.ms/azureml-infer-cli-image-text-embeddings" target="_blank">image-text-embeddings-online-endpoint.sh</a>
+Batch|<a href="https://aka.ms/azureml-infer-batch-sdk-image-text-embeddings" target="_blank">image-text-embeddings-batch-endpoint.ipynb</a>|<a href="https://aka.ms/azureml-infer-batch-cli-image-text-embeddings" target="_blank">image-text-embeddings-batch-endpoint.sh</a>
+
+### Sample inputs and outputs (for real-time inference)
```

Comment:
```suggestion
# Sample input and output (for real-time inference)
```

Reply:
Please remove: `(for real-time inference)` from the end as well